gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -650,14 +652,14 @@ public Response revoke(String token) {
     }
 
     if (error.isEmpty()) {
-      resp =  Response.status(Response.Status.OK)
-                      .entity("{\n  \"revoked\": \"true\"\n}\n")
-                      .build();
+      resp = Response.status(Response.Status.OK)
+              .entity("{\n  \"revoked\": \"true\"\n}\n")
```

Comment:
formatting change

Reply:
Not sure why it was the previous indentation. I guess someone is lining up the dots? That is not a convention that is standard.  That said, I don't know why this was changed at all. Likely some IDE shenanigans related to my migration to a new dev machine.